scm/etc/modules/hera_intel.lua

```diff
@@ -0,0 +1,46 @@
+help([[
+This module loads libraries for building the CCPP Single-Column Model on
+the NOAA RDHPC machine Hera using Intel-2022.1.2
+]])
+
+whatis([===[Loads libraries needed for building the CCPP SCM on Hera ]===])
+
+prepend_path("MODULEPATH","/contrib/sutils/modulefiles")
+load("sutils")
+
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env/install/modulefiles/Core")
```

Comment:
Can we use the same version of spack-stack as in ufs-weather-model: https://github.com/ufs-community/ufs-weather-model/blob/develop/modulefiles/ufs_hera.intel.lua?

Reply:
Yes - ufs will move to 1.5.1 shortly. Do you want me to create a PR and update to 1.5.1? This way you don't waste your time if something goes less smooth than I was bragging.

Reply:
@mkavulich Can you update this PR at least to spack-stack-1.5.0 and then once ufs moves to 1.5.1, we can try to follow suit. I'd like to keep the SCM using the same stack as ufs-wm going forward, we just haven't been paying attention for a while due to not having a release.